pkg/auth/dependency/userverify/sender.go

```diff
@@ -110,17 +110,17 @@ func prepareVerifyRequestContext(
 	userProfile userprofile.UserProfile,
 ) map[string]interface{} {
 	return map[string]interface{}{
-		"appname":      appName,
-		"record_key":   verifyCode.RecordKey,
-		"record_value": verifyCode.RecordValue,
-		"user_id":      userProfile.RecordID,
-		"user":         userProfile.ToMap(),
-		"code":         verifyCode.Code,
+		"appname":       appName,
+		"record_key":    verifyCode.RecordKey,
+		"record_value":  verifyCode.RecordValue,
+		"user_id":       userProfile.ID,
+		"user_metadata": userProfile.Data,
```

Comment:
I think the context for email template should include the whole user object for the key `user`, use case like displaying user created time or update time.

Same for other email template context.